java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -2070,8 +2070,6 @@ private static native long clamper(long nativeView, long loScalarHandle, long lo
    * r is now [false, true, false]
    * ```
    * Any null string entries return corresponding null output column entries.
-   * For supported regex patterns refer to:
-   * @link https://rapidsai.github.io/projects/nvstrings/en/0.13.0/regex.html
```

Comment:
Rather than remove this pointer to regex documentation, can we update it instead?  For example, https://github.com/rapidsai/cudf/blob/master/cpp/doxygen/regex.md could be useful here.  Maybe there's a better place to point users?  Comment applies to the other changes in this file.

Reply:
It looks like that gets rendered to https://docs.rapids.ai/api/libcudf/nightly/md_regex.html. Are we OK with pointing to that page? (cc: @jrhemstad who had some thoughts about better placing that page within our docs)

Reply:
yes

Reply:
You should be able to use a Doxygen command to link it rather than hard coding a URL that could change.

Reply:
I think it would be `@ref md_regex` or something like that.

Reply:
> I think it would be `@ref md_regex` or something like that.

Are the java docs doxygen and published in a way that meshes with our publishing of docs?

Reply:
The java docs are [java docs](https://en.wikipedia.org/wiki/Javadoc).  Because java is not built as a part of any regular build there is no publishing of them to anything rapids right now.  When we do a release a special documentation jar is created.  I'm not sure if it is pushed to maven central or not.  I need to check on it.

Reply:
You can do the @link like was originally there or you can just insert HTML and it works just fine.

Reply:
looks great